packages/gatsby-theme-docs/src/templates/release-notes-detail.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { MDXProvider } from '@mdx-js/react';
+import { Markdown } from '@commercetools-docs/ui-kit';
+import LayoutReleaseNotesDetail from '../layouts/release-notes-detail';
+import { SEO, ThemeProvider } from '../components';
+import markdownComponents from '../markdown-components';
+
+const releaseNoteMarkdownComponents = {
+  ...markdownComponents,
+  // NOTE: release notes content can only have headings starting from h4.
+  h1: Markdown.withAnchorLink(Markdown.H4),
+  h2: Markdown.withAnchorLink(Markdown.H5),
+  h3: Markdown.withAnchorLink(Markdown.H6),
+  h4: Markdown.withAnchorLink(Markdown.H6),
```

Comment:
This is the mapped list of headings for the release notes content.

cc @nkuehn